src/emulator/controller.ts

```diff
@@ -350,6 +353,17 @@ export async function startAll(options: any, noGui: boolean = false): Promise<vo
     await startEmulator(hostingEmulator);
   }
 
+  if (shouldStart(options, Emulators.LOGGING)) {
```

Comment:
Why would this ever not start?  Is this just a feature flag for testing or you think people would/shoudl control this?

Reply:
re the convo in the gui room, I do think there's reasons to not want it (it's basically shipping all local logs, assumed to be private, over a ws). I think we could default to it on, but there should be a choice I think

Reply:
Ok so I think we settled on `--except` support here and having this on by default?